src/coffee/run.coffee

```diff
@@ -203,6 +203,7 @@ module.exports = class
       .description 'Allows to publish, unpublish or delete (all) products of your SPHERE.IO project.'
       .option '--changeTo <publish,unpublish,delete>', 'publish unpublished products / unpublish published products / delete unpublished products'
       .option '--csv <file>', 'processes products defined in a CSV file by either "sku" or "id". Otherwise all products are processed.'
+      .option '-o, --output <file>', 'Path to the file if the product being process is more than the default which is 100.'
       .option '--continueOnProblems', 'When a there is a problem on changing a product\'s state (400er response), ignore it and continue with the next products'
       .option '--forceDelete', 'whether to force deletion without asking for confirmation', false
       .usage '--projectKey <project-key> --clientId <client-id> --clientSecret <client-secret> --changeTo <state>'
```

Comment:
since `-o` is now optional we don't need to add it to the example here :)

Reply:
For documentation purpose, it will be better to have it written down as part of the option available for `command-line usage` in the module. I'll modify the sentence to inform the user that it is an optional parameter.